elements/rh-secondary-nav/demo/rh-secondary-nav.html

```diff
@@ -0,0 +1,414 @@
+<link rel="stylesheet" href="https://patternflyelements.com/core/styles/pfe-layouts.min.css">
+<link rel="stylesheet" href="/elements/rh-secondary-nav/demo/demo.css"/>
+<link rel="stylesheet" href="/elements/rh-secondary-nav/rh-secondary-nav-lightdom.css"/>
+
+
+<pfe-band size="small" color="darkest">
+  <h2 slot="header">Note</h2>
+  <p>We dont have container queries yet, so this compnent's media query break points are set in understanding that the component will exist at full screen width.  The breakpoints will appear off if the component isn't 100% page width. You can use the fullscreen button above to test media query breakpoint.  However doing so will break position: sticky of the menu.</p>
+</pfe-band>
+
+<rh-secondary-nav role="navigation">
+  <a href="#" slot="logo" id="logo-id"> <!-- id is upgraded if not given -->
+    Red Hat Ansible Automation Platform <!-- <svg> or text can be added here -->
+  </a>
+  <ul slot="nav" aria-labelledby="logo-id"> <!-- is this ul labelledby the text or above what if its a svg? -->
+    <li>
+      <rh-secondary-nav-dropdown>
+        <a href="#" slot="link">Explore</a>
+        <rh-secondary-nav-menu slot="menu">
+          <rh-secondary-nav-menu-section slot="section">
+            <h3 slot="header" id="ansible-platform"> <!-- id will upgrade if not given -->
+              <a href="#">Why Red Hat Ansible Automation Platforms</a>
+            </h3>
+            <ul slot="links" aria-labelledby="ansible-platform""> <!-- aria-labelledby will upgrade if not given -->
+              <li>
+                <a href="#">
+                  Integrations
+                  <pfe-icon icon="web-icon-new-window" color="info" size="sm"></pfe-icon>
```

Comment:
beware eventual deprecation of pfe-icon or replacement with rh-icon

Reply:
`<pfe-icon>` in that case is light dom and could be replaced by anything.